helm_deploy/laa-estimate-eligibility/values/ccq-uat.yaml

```diff
@@ -0,0 +1,112 @@
+
+nodeSelector: {}
+
+tolerations: []
+
+affinity: {}
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: false
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: "cd-serviceaccount"
+
+podSecurityContext: {}
+# fsGroup: 2000
+
+service:
+  type: ClusterIP
+  port: 80
+
+securityContext:
+  runAsUser: 1000
+
+ingress:
+  hosts:
+#    - host: main-check-client-qualifies-legal-aid-uat.cloud-platform.service.justice.gov.uk
+    - host: main-ccq-uat.cloud-platform.service.justice.gov.uk
+      secret: false
+  annotations:
+    external-dns.alpha.kubernetes.io/set-identifier: "check-client-qualifies-laa-estimate-eligibility-laa-check-client-qualifies-uat-green"
+    # TODO: the weight is set to 0 for now but will need to be reverted to 100 at some point
+    external-dns.alpha.kubernetes.io/aws-weight: "0"
+
+notifications:
+  errorMessageTemplateId: b8879191-e4bb-4f10-831a-7d9bd2b078f1
+  recipient: patrick.gleeson@digital.justice.gov.uk,michael.blatherwick@digital.justice.gov.uk,william.clarke@digital.justice.gov.uk
+
+googleTagManager:
+  containerId: GTM-WJ98FR4
+
+cfe:
+  host: http://main-cfe-civil.cfe-civil-uat.svc.cluster.local
+  environment_name: uat
+
+featureFlags:
+  sentry: ENABLED
+  overrides: ENABLED
+  indexProduction: NOT_ENABLED
+  maintenanceMode: NOT_ENABLED
+  basicAuthentication: ENABLED
+
+postgresql:
+  enabled: true
+  resources:
+    limits:
+      cpu: 250m
+      memory: 256Mi
+    requests:
+      cpu: 250m
+      memory: 256Mi
+  auth:
+    database: ccq_uat
+    enablePostgresUser: true
+    postgresPassword: postgres
+  postgresqlUsername: postgres
+  primary:
+    persistence:
+      enabled: true
+
+geckoboard:
+  metricsDataset: "ccq_monthly_metrics_uat"
+  allMetricsDataset: "ccq_all_metrics_uat"
+  validationsDataset: "ccq_validations_uat"
+  lastPagesDataset: "ccq_last_pages_uat"
+  allJourneysDataset: "ccq_all_journeys_uat"
+  monthlyJourneysDataset: "ccq_monthly_journeys_uat"
+  recentJourneysDataset: "ccq_recent_journeys_uat"
+  enabled: not_enabled
+
+sentry:
+  # Despite looking very secret-y, the key here is _not_ a secret, as it will appear in every HTTP response header
+  cspReportEndpoint: https://o345774.ingest.sentry.io/api/6747538/security/?sentry_key=53f41c70bd59430cbf0855fd6f79cf86
+  # Similarly this is available in the page source
+  dsn: https://04ec9fc34a2146138f8a2da2d243a718@o345774.ingest.sentry.io/6747538
+
+app:
+  primaryHost: ""
+  seedAdmins: patrick.gleeson@digital.justice.gov.uk,michael.blatherwick@digital.justice.gov.uk,william.clarke@digital.justice.gov.uk,masum.khan@digital.justice.gov.uk,hettie.street@digital.justice.gov.uk
+
+google:
+  oauthClientId: 276075615370-ncqvmdi84jruv1jggfg4bu8nlvk1drj0.apps.googleusercontent.com
+  oauthRedirectUri: https://main-check-client-qualifies-legal-aid-uat.cloud-platform.service.justice.gov.uk/auth/subdomain_redirect
```

Comment:
This should probably be set to `main-ccq-uat.cloud-platform.service.justice.gov.uk`, since the other "main" host might go away with the other namespace. But if so, we need to whitelist `main-ccq-uat...` in the Google Developer Console as a valid redirect URI for OAuth.

Reply:
we were not 100% sure of the correct approach for this, will make those changes.

Reply:
Happy to make this change although I'm not sure how to do the whitelisting, if someone could help with that @patrick-laa @willc-work

Reply:
Sure @HettieS I think this link will get you to the right screen: https://console.cloud.google.com/apis/credentials/oauthclient/276075615370-ncqvmdi84jruv1jggfg4bu8nlvk1drj0.apps.googleusercontent.com?orgonly=true&project=ccq-admin&supportedpurview=organizationId - all you need to do is add a new entry to the list of 'Authorised redirect URIs' and save.

Reply:
thank you! this should now be done